static/docs/use-cases/data-and-model-files-versioning.md

```diff
@@ -23,9 +23,9 @@ ML <abbr>data artifacts</abbr> like data files, models, etc. Unlike `git-lfs`,
 DVC doesn't require installing a server; it can be used on-premises (NAS, SSH,
 for example) or with any major cloud provider (S3, Google Cloud, Azure).
 
-Let's say you already have a project that uses a bunch of images that are stored
-in `images` directory and has a `model.pkl` file - your model file that is
-deployed to production.
+Let's say you already have a <abbr>project</abbr> that uses a bunch of images
+stored in `images/` directory and has a `model.pkl` file - your model file that
```

Comment:
do we actually need to specify `/` after dir names? is it some common practice?

Reply:
I do it only in relative directory names that don't include other slashes `/` to make it extra clear that it's a dir and not a file. Thanks for reminding me actually, I just made it consistent throughout docs in 8650a84 but lmk if you'd prefer all file and dir names to NOT have `/` at the end.